apps/prairielearn/src/pages/instructorAssessmentSettings/instructorAssessmentSettings.html.ts

```diff
@@ -44,159 +44,215 @@ export function InstructorAssessmentSettings({
-                      <button
-                        name="__action"
-                        value="copy_assessment"
-                        class="btn btn-sm btn-primary"
-                      >
-                        <i class="fa fa-clone"></i> Make a copy of this assessment
-                      </button>
-                    </form>
+            <div class="card-body">
+            <form>
+              <div class="form-group">
+                <label for="title">Title</label>
+                <input
+                  type="text"
+                  class="form-control"
+                  id="title"
+                  name="title"
+                  value="${resLocals.assessment.title}"
+                  disabled
+                />
+                <small class="form-text text-muted"> The title of the assessment. </small>
+              </div>
+              <div class="form-group">
+                <label for="type">Type</label>
+                <input
+                  type="text"
+                  class="form-control"
+                  id="type"
+                  name="type"
+                  value="${resLocals.assessment.type}"
+                  disabled
+                />
+                <small class="form-text text-muted">
+                  The type of the assessment. This can be either Homework or Exam.
+                </small>
+              </div>
+              <div class="form-group">
+                <label for="set">Set</label>
+                <input
+                  type="text"
+                  class="form-control"
+                  id="set"
+                  name="set"
+                  value="${resLocals.assessment_set.name} (${
+                    resLocals.assessment_set.abbreviation
+                  })"
+                  disabled
+                />
+                <small class="form-text text-muted"> The set this assessment belongs to. </small>
+              </div>
+              <div class="form-group">
+                <label for="number">Number</label>
+                <input
+                  type="text"
+                  class="form-control"
+                  id="number"
+                  name="number"
+                  value="${resLocals.assessment.number}"
+                  disabled
+                />
+                <small class="form-text text-muted">
+                  The number of the assessment within the set.
+                </small>
+              </div>
+              <div class="form-group">
+                <label for="module">Module</label>
+                <input
+                  type="text"
+                  class="form-control"
+                  id="module"
+                  name="module"
+                  value="${resLocals.assessment_module ? resLocals.assessment_module.heading : ''}"
+                  disabled
+                />
+                <small class="form-text text-muted"> The course module this assessment has been assigned to. </small>
```

Comment:
Nit: set/module are basically the same, but described using different language:

- "set" vs "course module" (just call this module)
- "belongs to" vs. "has been assigned to" (I'd just use "belongs to" here.

Reply:
Updated language for module in [d81d796](https://github.com/PrairieLearn/PrairieLearn/pull/10653/commits/d81d79644453e2a6a9ed82bf367461c2804bc3ca).